### PR TITLE
Add create/update/delete monitors

### PR DESCRIPTION
**Pull request by evantissot**
### Changes
- Bootstrap `ansible-collection` repo which contains `toucantoco.toucantoco` custom Ansible collection (CI, structure, Build...) 
- Add a module `betteruptime_monitor` for managing tests (aka monitors in betteruptime vocabulary)

### CODE REVIEW

**Comment by evantissot** [comment]
[[Better Uptime] Monitor resources (nodes/instances) with Better Uptime](https://trello.com/c/nbmNOfJT/1160-better-uptime-monitor-resources-nodes-instances-with-better-uptime)

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,78 @@
+// This Jenkinsfile is dedicated to the jenkins job: ansible-collection
+// cf: https://jenkins.toucantoco.guru/job/ansible-collection/
```
Comment (ArcRiiad) [inline Jenkinsfile]: delete le comments

File: plugins/modules/betteruptime_monitor.py [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+        diff_attributes = {}
+        for key in self.payload:
+            if key == "request_headers":
+                same=all(i in [{"name": j["name"], "value": j["value"]} for j in self.retrieved_attributes[key]] for i in self.payload[key])
+                if not same or len(self.retrieved_attributes[key]) != len(self.payload[key]):
+                    destroyed_headers = [{"id": i["id"] , "_destroy": True} for i in self.retrieved_attributes[key]]
+                    self.payload[key] = self.payload[key] + destroyed_headers
+                    diff_attributes[key] = self.payload[key]
+            elif self.retrieved_attributes[key] != self.payload[key]:
+                diff_attributes[key] = self.payload[key]
```
Comment (ArcRiiad) [inline plugins/modules/betteruptime_monitor.py:115]: check quand on auras le temps si on peut simplifier cette partie

**Review by ArcRiiad — DISMISSED** [review]
Top la PR 🚀 

Ps: check l'histoire des for de for

File: plugins/modules/betteruptime_monitor.py (hunk 2 above)
Comment (ArcRiiad) [reply plugins/modules/betteruptime_monitor.py:115]: Faire une carte dans le backlog infra

File: Jenkinsfile (hunk 1 above)
Comment (evantissot) [reply Jenkinsfile]: Dropped :wastebasket:

File: plugins/modules/betteruptime_monitor.py (hunk 2 above)
Comment (evantissot) [reply plugins/modules/betteruptime_monitor.py:115]: Done :white_check_mark:

**Review by ArcRiiad — APPROVED** [review]

**Pull request merged**